dockerfile-generator.rb

```diff
@@ -0,0 +1,392 @@
+#!/usr/bin/env ruby
```

Comment:
keeping the same format and the `&&` and `\` at the end. if you want me to just update the `Dockerfile` I'm ok with that I can update it now.